active/tab/tabcontrol.md

```diff
@@ -0,0 +1,252 @@
+
+``` xml
+<TabControl CanDragItems="True"
+            CanReorderItems="True"
+            TabDraggedOutside="TabControl_TabDraggedOutside">
+```
+``` csharp
+// NOTE: The app is responsible for writing this code. We will provide a sample that may look something like:
+private async void TabControl_TabDraggedOutside(object sender, TabDraggedOutsideEventArgs e)
+{
+    // Create a new AppWindow
+    AppWindow newWindow = await AppWindow.TryCreateAsync();
+
+    // Create the content for the new window
+    var newPage = new MainPage();
+
+    // Remove tab from existing list
+    Tabs.Items.Remove(e.Tab);
+
+    // Add tab to list of Tabs on new page
+    newPage.AddItemToTabs(e.Tab);
+
+    // Set the Window's content to the new page
+    ElementCompositionPreview.SetAppWindowContent(newWindow, newPage);
+
+    // Show the window
+    await newWindow.TryShowAsync();
+}
+```
+
+## Databind to a set of tabs 
+
+``` xml
+<TabControl ItemsSource="{x:Bind TabItemCollection}" />
+```
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples
+section.  For example, see the Remarks for the MediaPlayerElement 
+(https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.MediaPlayerElement#remarks). -->
+
+## Keyboarding behavior
+### TAB and Arrow key behavior
+The TabControl has areas containing left content, right content, and the Tab content (containing 0 or more Tabs). When the user hits the TAB key, focus moves between the content areas. When focus moves into the Tab content area, the selected Tab gains focus. Once the user is in the Tab content area, pressing the arrow key will move focus between the tabs. Arrow focus is trapped inside the Tab content area.
+
+### Selecting a Tab
+When a Tab has focus, pressing SPACE or ENTER will select that Tab.
+
+### Shortcuts for moving between tabs
+Ctrl+TAB will select the next Tab.
+Ctrl+Shift+TAB will select the previous Tab.
+Tab selection cycles (meaning if the user has selected the last tab and presses Ctrl+TAB, the first tab becomes selected).
+
+Ctrl+1 through 8 will automatically select the corresponding Tab.
+Ctrl+9 will automatically select the last Tab.
+
+### Closing a Tab 
+Hitting CTRL+W or CTRL+F4 will automatically close the selected Tab (assuming it is closable). 
+
+### Keyboard guidance for App Developers
+The above sections outline built-in keyboarding behavior provided by the TabControl. However, there are certain expected keyboard shortcuts that you will be responsible for implementing. 
+* If your Tab control supports adding a new Tab, users expect CTRL+T to open a new tab
+* Consider maintaining a list of recently closed Tabs. If the user presses CTRL+SHIFT+T, they expect recently closed tabs to be reopened.
+* If users can perform more commands on a Tab than just closing the Tab (for example, pinning a Tab or duplicating a Tab), consider adding a context menu to the Tab. 
```

Comment:
For an app to be able to pin a TabItem there are things that may be wanted that aren't covered in this spec.
- Can TabItem order be changed programmaticly? (As the underlying type for the Items isn't specified I don't want to assume.)
- Can the ability to reorder individual Items be prevented? (So can do things like put all the pinned tabs together.)